package.json

```diff
@@ -136,10 +135,11 @@
     "@nrwl/nx-plugin": "10.0.7",
     "@nrwl/workspace": "10.0.7",
     "@octokit/rest": "^15.17.0",
+    "@testing-library/cypress": "^7.0.1",
```

Comment:
I installed Testing Library Cypress because all the e2e tests were failing.
I didn't dig too much into this, but the culprit was that placeholders weren't added to the controls, but it was using the attribute `data-placeholders` . 
I wasn't able to reproduce this issue in a StackBlitz...